clinica/iotools/converters/genfi_to_bids/genfi_to_bids_utils.py

```diff
@@ -835,7 +852,7 @@ def _get_multipart_id(nb_parts: PhilipsNumberOfParts, run_num: str) -> Optional[
             return "dwi_1"
         raise ValueError(f"{run_num} is outside of the scope.")
     if nb_parts == PhilipsNumberOfParts.FIVE:
-        if run_num in (f"run-0{k}" for k in range(1, 5)):
+        if run_num in (f"run-0{k}" for k in range(1, 6)):
```

Comment:
Was the 5 a mistake or has something changed ?

Reply:
No ! We want to have the 5 first runs